src/main/java/pl/tomaszdziurko/jvm_bloggers/JvmBloggersConfiguration.java

```diff
@@ -33,4 +34,8 @@ public Client getMailingRestClient(@Value("${mailing.apiKey}") String malingApiK
         return client;
     }
 
+    @Bean
+    public CacheManager cacheManager() {
+        return new GuavaCacheManager();
```

Comment:
Hmm.. we are not on the same page here :) Cache is mostly used to store/retrieve data. Not sure what are you referring to :)